meshing/src/CalculateNormals.cpp

```diff
@@ -38,21 +60,37 @@ void CalculateNormals::compute_normals_()
 
         // To-Do: #185
 
+        ITKPoint facePoint;
```

Comment:
Long term goal is to minimize use of ITK, so I'd prefer this to be a `cv::Vec3d`. This can make some stuff later easier (which I'll indicate). 

Also, `centroid` might be a better variable name.